### ISSUE TITLE
always optional type

### ISSUE THREAD

**Issue by alex-shatalov** [acacode/swagger-typescript-api#68]
Hi, i have schema
```yaml
  EventsList:
    properties:
      rows:
        items:
          $ref: '#/definitions/EventRow'
        type: array
        x-omitempty: false
      items:
        items:
          $ref: '#/definitions/Event'
        type: array
        x-omitempty: false
      total:
        format: int64
        type: integer
        x-omitempty: false
      tableFormat:
        $ref: '#/definitions/TableFormat'
        x-omitempty: false
    type: object
```

and i am getting

```ts
export interface EventsList {
  rows?: EventRow[];
  items?: Event[];
  total?: number;
  tableFormat?: TableFormat;
}
```

is it somehow possible to get strick type? we also try x-nullable: false and it works, but x-omitempty no.

**Comment by js2me** [comment]
@alex-shatalov hello, thanks for the issue!   
Which schema version do you use?

**Comment by alex-shatalov** [comment]
v2

**Comment by js2me** [comment]
@alex-shatalov That's definitely a bug. I'll fix that   
For now, as a trick you can add `required` keyword to the definition:   
```yaml
  EventsList:
    properties:
      rows:
        items:
          $ref: '#/definitions/EventRow'
        type: array
      items:
        items:
          $ref: '#/definitions/Event'
        type: array
      total:
        format: int64
        type: integer
      tableFormat:
        $ref: '#/definitions/TableFormat'
    type: object
    required:
      - rows
      - items
      - total
      - tableFormat
```

**Comment by alex-shatalov** [comment]
@js2me thanks a lot!!

**Comment by js2me** [comment]
Fixed in 1.12.0!
